src/translations/en.json

```diff
@@ -2440,7 +2440,10 @@
                   "mode_below": "Below mode",
                   "value_template": "Value template",
                   "type_value": "Fixed number",
-                  "type_input": "Numeric value of another entity"
+                  "type_input": "Numeric value of another entity",
+                  "description": {
+                    "full": "When {hasAttribute, select, \n true {{attribute} from } \n other {}\n }{entity} is {hasAboveAndBelow, select, \n above {above {above}} \n aboveAndBelow {above {above} and below {below}}\n below {below {below}}\n other {}\n }{hasDuration, select, \n true { for {duration}} \n other {}\n }"
```

Comment:
I feel like this is becoming a bit too complicated for the translators now, can we split this in to multiple keys?

Reply:
In the earlier pull requests you asked not to do so because the place of the keys can be different in other languages. Things will get more complicated with the left over triggers and conditions, so what keys would you suggest for this one?

Reply:
I would not choose to split the sentence, but create separate keys for the different use cases. It will give redundant translations, but is easier for translators I think... (so... both are not ideal) they will still stay pretty complicated anyway... 🤷‍♂️ 

Maybe:
```json
{
	"below": "When {hasAttribute, select, \n true {{attribute} from } \n other {}\n }{entity} is above {above} {hasDuration, select, \n true { for {duration}} \n other {}\n }",
	"above": "When {hasAttribute, select, \n true {{attribute} from } \n other {}\n }{entity} is below {below} {hasDuration, select, \n true { for {duration}} \n other {}\n }",
	"above-below": "When {hasAttribute, select, \n true {{attribute} from } \n other {}\n }{entity} is above {above} and below {below} {hasDuration, select, \n true { for {duration}} \n other {}\n }",
}
```

Reply:
After rereading the docs, I don't think there are other possibilities. I've just read something about undefined, which I've also applied on these strings now. 

I've splitted them as you suggested and they are easier to read now, thank you!